report_thesis/src/sections/background/linear_and_regularization_models/lasso.tex

```diff
@@ -0,0 +1,20 @@
+\subsubsection{Least Absolute Shrinkage and Selection Operator (LASSO) Regression}
+Having introduced ridge regression, we now turn our attention to the \gls{lasso} regression based on \citet{James2023AnIS}.
+
+\gls{lasso} regression addresses the same issue of multicollinearity as ridge regression but does so by introducing a different regularization term to the \gls{olsr} objective function.
+While ridge regression uses the $L_2$ norm, \gls{lasso} uses the $L_1$ norm, which has the distinct effect of performing feature selection by shrinking some coefficients to exactly zero:
```

Comment:
Perhaps a few words on how shrinking the coeffecients leads to feature selection. Just so we clearly demonstrate that we actually know how that works.